acis_thermal_check/utils.py

```diff
@@ -183,48 +183,67 @@ def get_options(msid, name, model_path, opts=None):
     from argparse import ArgumentParser
     parser = ArgumentParser()
     parser.set_defaults()
-    parser.add_argument("--outdir", default="out", help="Output directory")
-    parser.add_argument("--oflsdir", help="Load products OFLS directory")
+    parser.add_argument("--outdir", default="out", help="Output directory. Default: 'out'")
+    parser.add_argument("--backstop_file", help="Path to the backstop file. If a directory, "
+                                                "the backstop file will be searched for within "
+                                                "this directory. Default: None")
+    parser.add_argument("--oflsdir", help="Path to the directory containing the backstop "
+                                          "file (legacy argument). If specified, it will "
+                                          "override the value of the backstop_file "
+                                          "argument. Default: None")
     parser.add_argument("--model-spec", 
                         default=os.path.join(model_path, '%s_model_spec.json' % name),
-                        help="Model specification file")
+                        help="Model specification file. Defaults to the one included with "
+                             "the model package.")
     parser.add_argument("--days", type=float, default=21.0,
-                        help="Days of validation data (days)")
-    parser.add_argument("--run-start", help="Reference time to replace run "
-                                            "start time for regression testing")
-    parser.add_argument("--traceback", default=True, help='Enable tracebacks')
+                        help="Days of validation data. Default: 21")
+    parser.add_argument("--run-start", help="Reference time to replace run start time "
+                                            "for regression testing. The default is to "
+                                            "use the current time.")
+    parser.add_argument("--interrupt", help="Set this flag if this is an interrupt load.",
```

Comment:
Should this be determined from products instead of supplied via cmdline?

Reply:
The run-start option has always been there, it’s for testing, and if not specified it is determined from products. There is no change to this behavior from what it was doing previously.

AFAIK there is no way from the products to determine if the load is an interrupt of some kind—the ACIS load review software has always had a flag for interrupts that had to be supplied by the person running the tool manually. The only change here is that we are passing it to the thermal model (which wasn’t done before)

Reply:
Right, my comment was on `--interrupt`.  And I didn't know your review software already expected this manually so good to know.

In the processing we do to make the timelines table, I think I use the file in products called the "Command Load Generation Processing Summary" (see 2017/AUG3017/oflsa/mps/C243_1508.sum for an example) to figure out if the products in review are replan-reopen or replan-interrupt.  But I think that development was done without a spec on the content of that file.  The fact that there isn't a standard way to see this might also speak to the continued need for better initial conditions in a machine-readable form in products.